contract.go

```diff
@@ -47,7 +47,7 @@ func readSummaryFromModulefile(cmd *directoryModule) (string, error) {
 			exit.Wrap(
 				CommandSummaryError{
 					CommandError{
-						Message: fmt.Sprintf("error reading %s: %s", cmd.path, err),
+						Message: fmt.Sprintf("summary('%s'): %s", Usage(cmd), err),
```

Comment:
fewer characters + `Usage(cmd)` is more useful than `cmd.path` (especially for multi-clis that provide a bunch of commands from one binary)